src/libs/const.js

```diff
@@ -0,0 +1,35 @@
+const MESSAGE = {
+  start: '숫자 야구 게임을 시작합니다.',
+  success: '3개의 숫자를 모두 맞히셨습니다! 게임 종료',
+  inputNumber: '숫자를 입력해주세요 : ',
+  inputOption: '게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.\n',
+};
+
+const ERROR_MESSAGE = {
+  isNotNumber: '[ERROR] 숫자만 입력 가능합니다.',
+  length: '[ERROR] 숫자 3개를 입력해야 합니다.',
+  includesZero: '[ERROR] 0은 입력할 수 없습니다.',
+  overlap: '[ERROR] 중복된 숫자를 입력할 수 없습니다.',
+  option: '1 또는 2를 입력하지 않아 애플리케이션이 종료됩니다.',
```

Comment:
에러의 머릿말을 분리시키면 더 좋은 확장성과 명시성을 가지게 될 것 같습니다 :)

```suggestion
  errorHeading: '[ERROR]'
  isNotNumber: '숫자만 입력 가능합니다.',
  length: '숫자 3개를 입력해야 합니다.',
  includesZero: '0은 입력할 수 없습니다.',
  overlap: '중복된 숫자를 입력할 수 없습니다.',
  option: '1 또는 2를 입력하지 않아 애플리케이션이 종료됩니다.',
```